_episodes/04-recipe.md

```diff
@@ -330,331 +73,545 @@ Please note the following sections:
+> >   which contains preprocessed data for each of the input datasets, plus a
+> >   combined 'MultiModelMean', and `metadata.yml`.
+> > - **map/script1**: creates `/run/map/script1` with general information and a
+> >   log of the diagnostic script run. It also creates `/plots/map/script1` and
+> >   `/work/map/script1`, which contain output figures and output datasets,
+> >   respectively. For each output file, there is also corresponding provenance
+> >   information in the form of `.svg`, `.xml`, `.bibtex` and `.txt` files.
+> > - **timeseries/script1**: creates `/run/timeseries/script1` with general
+> >   information and a log of the diagnostic script run. It also creates
+> >   `/plots/timeseries/script1` and `/work/timeseries/script1`, which contain
+> >   output figures and output datasets, respectively. For each output file,
+> >   there is also corresponding provenance information in the form of `.svg`,
+> >   `.xml`, `.bibtex` and `.txt` files.
+> >
+> {: .solution}
+{: .challenge}
 
-> ## The diagnostic log file.
+> ## Pro tip: diagnostic logs
 >
-> This file is located at `./recipe_example_#_#/run/diag_timeseries_temperature/timeseries_diag/log.txt`
-> Let's have look at the file:
+> When you run ESMValTool, any log messages from the diagnostic script are not printed on the terminal.
+> But they are written to the `log.txt` files in the folder `/run/<diag_name>/log.txt`.
 >
-> ~~~bash
-> nano ./recipe_example_#_#/run/diag_timeseries_temperature/timeseries_diag/log.txt
-> ~~~
->
-{: .solution}
-
-## Do your first edits
+> ESMValTool *does* print a command that can be used to re-run a diagnostic script. When you use this
+> the output *will* be printed to the command line.
+{: .callout}
 
-> ## Edit the recipe and run again
->
-> So far, the example recipe has used global volume-weighted ocean temperature.
-> Please edit this recipe to investigate one of the following fields:
->
-> - Land surface temperature (ts) for dataset HadGEM2-ES for 1970-2000
-> - Atmospheric surface average temperature (tas) for dataset HadGEM2-ES for 1970-2000
-> - Ocean surface average temperature (tos) for datasets HadGEM2-ES and HadGEM2-CC for 1970-2000
->
-> You will need to edit:
->
-> - the dataset:
->
->   - `mip`, `start_year`, `end_year`
->
-> - the preprocessor:
->
->   - These fields are all 2D fields, but thetaoga was a 0D field. This means
->     that we need to take the average over the latitude and longitude
->     dimensions. To do this, add the area_statistics to the preprocessor.
->
-> - the diagnostic:
->
->  - change the short_name value (thetaoga) for another:
->
->     - Land surface average temperature (ts)
->     - Atmospheric surface average temperature (tas)
->     - Ocean surface average temperature (tos)
->
+## Modifying the example recipe
+
+Let's make a small modification to the example recipe.
+
+> ## Change your location
+>
+> Modify the recipe to analyse the temperature for your own location.
```

Comment:
I suggest to add a note here to run the local file, i.e. `esmvaltool run example_python.yml` so that they do not use the previous command and wonder why it does not work.